docs/understanding-tracking-design/managing-tracking-scenarios-api/index.md

```diff
@@ -0,0 +1,367 @@
+            "type": "object",
+            "properties": {
+                "tracked": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "source": { "type": "string", "format": "uri" },
+                            "minCardinality": {
+                                "type": "integer",
+                                "minimum": 0,
+                                "default": 0
+                            },
+                            "maxCardinality": { "type": "integer" }
+                        },
+                        "required": ["source"]
+                    }
+                },
+                "enriched": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "source": { "type": "string", "format": "uri" },
+                            "minCardinality": {
+                                "type": "integer",
+                                "minimum": 0,
+                                "default": 0
+                            },
+                            "maxCardinality": { "type": "integer" }
+                        },
+                        "required": ["source"]
+                    }
+                }
+            }
+        }
+    },
+    "required": ["name", "event"]
+}
+```
+
+- **Name**: It validates that the `scenario.name` of a tracking scenario is unique within the data structure inferred from the `scenario.event.source`.
+- **Entities**: If present during creation, it will validate that the entities, `scenario.entities.tracked` and `scenario.entities.enriched`, are not duplicated.
+
+### Compatibility Check
+
+If a `scenario.event` is provided during creation, a compatibility check will be performed between the provided event and the one referenced in Iglu by `scenario.event.source`.
+
+This check is performed for the version indicated by the Iglu URI, referred to as the **current** version. Additionally, the compatibility check is also run for the **latest** version. The compatibility check results in three possible values: **Compatible**, **SchemaIncompatible**, or **SchemaUndecidable**. The different errors will be provided in the `errors` field of the [response](#response-format).
+
+- If **Compatible**, no errors or warnings will be attached to the `errors` response field.
+- If **SchemaIncompatible**, an error will be attached to the `errors` response field when the **current** version is incompatible, resulting in a **422 Unprocessable Entity** response without being able to create the tracking scenario. If the **latest** version is incompatible, a warning will be attached to the `errors` field, but a **201 Created** response will still be returned.
+- If **SchemaUndecidable**, a warning will be attached to the `errors` response field.
```

Comment:
I'm wondering if, at this point, a citation of the paper we based all this work on would be useful.

Reply:
Agreed, good point. I added a reference to the paper.